_docs/concepts/load-balancing.md

```diff
@@ -28,6 +37,12 @@ round robin, random, and weighted least request.
 In addition to load balancing, Envoy periodically checks the health of each
 instance in the pool. Envoy follows a circuit breaker style pattern to
 classify instances as unhealthy or healthy based on their failure rates for
-the the health check API call. In other words, when the number of health
+the health check API call. In other words, when the number of health
 check failures for a given instance exceeds a pre-specified threshold, it
-will be ejected from the load balancing pool.
+will be ejected from the load balancing pool. Similarly, when the number of
```

Comment:
Some places you use a dash in load-balancing, some places you don't. Let's pick one and add it to our list of authoring term guidelines that Louis started.